### PR TITLE
Add cookie notice

### PR DESCRIPTION
**Pull request by JoelPasvolsky**
@randomir, can you take a quick look so I can do the same for dwave_tabu, thanks!

### CODE REVIEW

File: docs/_static/cookie_notice.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  cursor: pointer;
+  padding-top: 12px;
+  padding-bottom: 12px;
+  padding-left: 42px;
+  padding-right: 42px;
+  margin: 1em;
+  margin-left: 2em;
+  white-space: nowrap;
+  vertical-align: middle;
+}
```
Comment (randomir) [inline docs/_static/cookie_notice.css]: This is Leap theme banner on RtD theme. It just doesn't feel right. :smile:

File: docs/_static/cookie_notice.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  ga('create', 'UA-124167090-6', 'auto');
+  ga('send', 'pageview');
+}
+
+let cookieAccepted = localStorage.cookieAccepted;
+if (cookieAccepted == undefined) {
+  $(document).ready(function () {
+    var cookieDiv = document.createElement('div')
+    cookieDiv.id = 'cookie-notice'
+    cookieDiv.innerHTML = "<p style='margin: 1em'>Privacy and Cookies: This site uses cookies. By continuing to use this website, you agree to their use. To find out more, see our <a style='color:#4c71c6; text-decoration: underline;' href='https://cloud.dwavesys.com/leap/legal/privacy_policy/' target='_blank'>Privacy Policy</a>.<button id='cookie-accept' class='cookie-button'>Close and Accept</button></p>"
```
Comment (randomir) [inline docs/_static/cookie_notice.js]: We're actually using Google Analytics, and Google is storing the cookie, not DWave.

The privacy policy (used on Leap) does mention sharing of user's data with 3rd party services that act on our behalf, but should here actually Google ask for user's permission, and say they'll share the analytics with DWave?

File: docs/_static/cookie_notice.css (hunk 1 above)
Comment (randomir) [reply docs/_static/cookie_notice.css]: Btw, to use something closer to RtD, it's enough to change the accept button. A few lines:
```
-  font-size: 14px;
+  font-size: 16px;

-  background: radial-gradient(circle, #4C71C6 0%, #002B86 100%);
+  background: rgb(41, 128, 185);

-  text-transform: uppercase;
-  letter-spacing: 1.2px;
```

Comment (JoelPasvolsky) [reply docs/_static/cookie_notice.css]: Will change, thanks

**Comment by JoelPasvolsky** [comment]
@randomir, I think this can be merged now, thanks.

**Pull request merged**